_includes/_landing_header.html

```diff
@@ -1,18 +1,10 @@
 <div class="site_header_landing site_header_img ">
-	<header class="site_header">
-	  <div class="container">
-	    <h1 class="ots_logo">
-	      <a href="{{site.baseurl}}">OpenTechSchool</a>
-	    </h1>
-	    <p>Tech Education how it is supposed to be: Hands-on, empowering, peer-driven.</p>
```

Comment:
If you check the image i attach, the slogan isn't gone, is just moved in better place. before it was very hard to read since was over the image